tests/test_merkle.py

```
@@ -0,0 +1,96 @@
+        txs = [
+            {"sender": "alice", "receiver": "bob", "amount": 10},
+            {"sender": "bob", "receiver": "charlie", "amount": 5}
+        ]
+        tree = MerkleTree(txs)
+        
+        wrong_root = "0" * 64
+        tx_hash = tree.tx_hashes[0]
+        proof = tree.get_proof(0)
+        
+        result = MerkleTree.verify_proof(tx_hash, proof, wrong_root)
+        self.assertFalse(result)
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertFalse`

Replace `assertFalse(...)` with `assert ...`

(PT009)